cypress/e2e/data-import/make-fields-required-in-invoice-map-profile.cy.js

```diff
@@ -5,7 +5,7 @@ import FieldMappingProfiles from '../../support/fragments/data_import/mapping_pr
 import NewFieldMappingProfile from '../../support/fragments/data_import/mapping_profiles/newFieldMappingProfile';
 import Helper from '../../support/fragments/finance/financeHelper';
 import Users from '../../support/fragments/users/users';
-import MappingProfileDetails from '../../support/fragments/data_import/mapping_profiles/mappingProfileDetails';
+import MappingProfileDetailsView from '../../support/fragments/data_import/mapping_profiles/mappingProfileDetailsView';
```

Comment:
Unable to resolve path to module '../../support/fragments/data_import/mapping_profiles/mappingProfileDetailsView'
I think it should be '../../support/fragments/data_import/mapping_profiles/fieldMappingProfileView'. Check please

Reply:
Done